### DISCUSSION TITLE
Waline评论插件添加失败

### DISCUSSION THREAD

**Discussion by nanbowanya** [ppoffice/hexo-theme-icarus#879] [Q&A] [answered]
![](https://tva1.sinaimg.cn/large/008i3skNgy1gshzerrqh6j30wi0kaq9n.jpg)
作者你好，我在添加Waline评论插件时提示如上，请问是否有详细教程可以帮助配置。

**Comment by ppoffice** [comment, marked as answer]
@nanbowan323 Waline在[hexo-component-inferno@0.12.0](https://github.com/ppoffice/hexo-component-inferno/releases/tag/0.12.0)中添加，对应主题版本为[4.3.0](https://github.com/ppoffice/hexo-theme-icarus/releases/tag/4.3.0)。请检查你是否升级到了最新版本。

**Reply by nanbowanya** [reply]
感谢，是版本问题
